supportedBackends/sfs/src/test/scala/cromwell/backend/sfs/SharedFileSystemSpec.scala

```diff
@@ -141,6 +142,33 @@ class SharedFileSystemSpec extends FlatSpec with Matchers with Mockito with Tabl
     dests.foreach(_.delete(swallowIOExceptions = true))
   }
 
+it should "copy the file again when the copy-cached file has exceeded the maximum number of hardlinks" in {
+    val callDirs: List[Path] = List(1,2,3).map(x => DefaultPathBuilder.createTempDirectory("SharedFileSystem"))
```

Comment:
```suggestion
    val callDirs: List[Path] = 1 to 3 foreach { _ => DefaultPathBuilder.createTempDirectory("SharedFileSystem") }
```

Reply:
:+1: . I had to change the type annotation of `callDirs` to `indexedSeq[Path]` but it works nicely.
And using the x unnecessaily was indeed an oversight. Thanks for spotting.